### PR TITLE
INS-2439: Move GetDelegate message handling to flow

### PR DESCRIPTION
**Pull request by bigbes**

### CODE REVIEW

File: ledger/light/handle/get_delegate.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+func (s *GetDelegate) Present(ctx context.Context, f flow.Flow) error {
+	msg := s.parcel.Message().(*message.GetDelegate)
+
+	jet := proc.NewFetchJet(*msg.DefaultTarget().Record(), flow.Pulse(ctx), s.replyTo)
+	s.dep.FetchJet(jet)
+	if err := f.Procedure(ctx, jet, false); err != nil {
+		return err
+	}
+
+	idx := proc.NewGetIndex(msg.Head, jet.Result.Jet, s.replyTo, s.parcel.Pulse())
```
Comment (andreyromancev) [inline ledger/light/handle/get_delegate.go]: flow.Pulse(ctx)

File: ledger/light/handle/get_delegate.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		dep:     dep,
+		parcel:  parcel,
+		replyTo: rep,
+	}
+}
+
+func (s *GetDelegate) Present(ctx context.Context, f flow.Flow) error {
+	msg := s.parcel.Message().(*message.GetDelegate)
+
+	jet := proc.NewFetchJet(*msg.DefaultTarget().Record(), flow.Pulse(ctx), s.replyTo)
```
Comment (andreyromancev) [inline ledger/light/handle/get_delegate.go]: msg.Head.Record()

Comment (bigbes) [reply ledger/light/handle/get_delegate.go]: fixed

File: ledger/light/handle/get_delegate.go (hunk 1 above)
Comment (bigbes) [reply ledger/light/handle/get_delegate.go]: fixed

**Pull request merged**